text-icu.cabal

```diff
@@ -46,6 +46,11 @@ license:        BSD3
 license-file:   LICENSE
 build-type:     Simple
 
+flag homebrew
+  Description: Assume homebrew on macOS. Automatically add /usr/local/opt/ and /usr/homebrew/opt/ paths to extra-lib-dirs and include-dirs.
```

Comment:
> /usr/homebrew/opt/

That meant `/opt/homebrew/opt/`, I suppose.

Reply:
Yes, I fixed it.